taker-frontend/src/components/Trade.tsx

```diff
@@ -221,7 +221,7 @@ export default function Trade({
                                     </Td>
                                 </Tr>
                                 <Tr>
-                                    <Td>Funding Rate</Td>
+                                    <Td>Perpetual Cost</Td>
                                     <Td isNumeric>{fundingRateHourly ? fundingRateHourly + "%" : "not available"}</Td>
```

Comment:
This number will flip its sign depending on who gets paid right? Is "cost" the best word in both cases?

Probably subject to a larger discussion, I am okay with moving forward to align it for now and then change it later in multiple places.

Reply:
I think this should be cleaned up together in #1692